### PR TITLE
Update a few deps, fix issues with security audit

### PR DESCRIPTION
**Pull request by jp-andre**

### CODE REVIEW

File: src/commands/distribute/groups/download.ts [hunk 1]
```diff
@@ -148,37 +148,38 @@ export default class DownloadBinaryFromDistributionGroupCommand extends AppComma
...
+      }
     } else {
       // using current working directory by default
       return Promise.resolve(cwd());
     }
   }
 
-  private getFileFullPath(passedFileName: string, directoryPath: string, appName: string, downloadUrl: string): string {
+  private getFilenameFromDownloadUrl(downloadUrl: string): string {
+    const filename = Url.parse(downloadUrl).pathname?.split("/").slice(-1)[0];
```
Comment (anywherepilot) [inline src/commands/distribute/groups/download.ts]: Won't this select just a single character? The `slice(-1)` selects the last string from the array of strings, and then `[0]` selects the first character of that string.

File: src/commands/distribute/groups/download.ts [hunk 2]
```diff
@@ -148,37 +148,38 @@ export default class DownloadBinaryFromDistributionGroupCommand extends AppComma
...
+    debug(`Got filename from URL: ${filename}`);
+    return filename;
+  }
+
+  private getFileFullPath(passedFileName: string, directoryPath: string, downloadUrl: string): string {
```
Comment (anywherepilot) [inline src/commands/distribute/groups/download.ts]: I'm sure these changes are fine, but how are they related to the dependency change?

File: typings/index.d.ts [hunk 3]
```diff
@@ -1,4 +1,3 @@
 /// <reference path="modules/azure-storage/index.d.ts" />
```
Comment (anywherepilot) [inline typings/index.d.ts]: Why are/were files like this in the repo?

File: src/commands/distribute/groups/download.ts (hunk 1 above)
Comment (anywherepilot) [reply src/commands/distribute/groups/download.ts]: Ah no I'm stupid ^^ Slice returns another array of course. Can't you take `[-1]` from the array? Is that Ruby?

Comment (jp-andre) [reply src/commands/distribute/groups/download.ts]: You can't do [-1]. I asked StackOverflow for the one-liner.

File: src/commands/distribute/groups/download.ts (hunk 2 above)
Comment (jp-andre) [reply src/commands/distribute/groups/download.ts]: Good catch! This is an overdue fix for the new ACFUS style URLs.

**Pull request merged**